### PR TITLE
fix(@desktop/communities): remove context menu leftover

### LINKED ISSUE #2839: Remove context menu leftover if no actions are applicable in community channel / category
1. create a community with channel as admin 
2. invite a contact to community
3. as invitee, go to the channel and right click an area under the channel name

Since no actions are applicable to non-admin in this case -> remove the context menu leftover for channels and categories

Screenshot 2021-07-07 at 16 20 35

### PR DESCRIPTION
**Pull request by alaibe**
fixes #2839

depends of: https://github.com/status-im/StatusQ/pull/268

image

Right click are disabled, as well as + action and category action

### CODE REVIEW

File: ui/app/AppLayouts/Chat/CommunityColumn.qml [hunk 1]
```diff
@@ -181,7 +178,6 @@ Item {
                 }
 
                 StatusMenuItem {
-                    enabled: chatsModel.communities.activeCommunity.admin
```
Comment (PascalPrecht) [inline ui/app/AppLayouts/Chat/CommunityColumn.qml:184]: Are we sure about these removals? Should anybody be able to edit an delete categories? How is that related to removing the context menu leftover?

Comment (PascalPrecht) [reply ui/app/AppLayouts/Chat/CommunityColumn.qml:184]: Ah I see, I guess this is because the menu will only be shown when the user is an admin anyways, so the condition here can be removed.

Comment (alaibe) [reply ui/app/AppLayouts/Chat/CommunityColumn.qml:184]: Exactly, before the check was on each item with the same condition, that is why I was able to move the check to the parent and avoid the duplication of the check for each entry.

**Review by jrainville — APPROVED** [review]
